### DISCUSSION TITLE
How to make the bot join more than one server

### DISCUSSION THREAD

**Discussion by ThisJackRN** [PrismarineJS/mineflayer#1832] [Q&A]
So I want to put my bot on more than one server right now but have zero Idea how could anyone give me a link to it?

**Comment by Caleb-Tan** [comment]
> So I want to put my bot on more than one server right now but have zero Idea how could anyone give me a link to it?

This is impossible with just one account. You would need to have multiple bot accounts to log into more than one server concurrently.

**Reply by ThisJackRN** [reply]
I got it fixed anyways thanks for the help
